nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
I think there has been a "handshake agreement" on the nature of the framework code versus the extension developer facing API.  Having said that, I was not able to find any documentation that indicates that is the case.  As a result, I think I am a similar 95/5 split as Tony.  As a result, I think the "right" thing given the circumstances is to tag it up as Deprecated and remove at 1.x.  However, I think it's likely a fair assumption that this is clear.  Either way, we need to get the handshake agreement in a tangible form some place.  I'll make a ticket and please add on/enhance as needed.